src/clr-angular/popover/dropdown/dropdown-trigger.ts

```diff
@@ -42,8 +45,24 @@ export class ClrDropdownTrigger {
     return this.ifOpenService.open;
   }
 
+  get setAriaLabel() {
```

Comment:
nit, naming is a little strange to me for a property, I'd like this to just be `ariaLabel` since this method is not setting anything directly.